backend/core/src/listings/dto/listing.dto.ts

```diff
@@ -807,7 +807,25 @@ export class ListingFilterParams extends BaseFilter {
     example: "true",
     required: false,
   })
-  [ListingFilterKeys.seniorHousing]?: boolean
+  [ListingFilterKeys.seniorHousing]?: boolean;
+
+  @Expose()
+  @ApiProperty({
+    type: Number,
+    example: "300",
+    required: false,
+  })
```

Comment:
Can you add `@IsOptional({ groups: [ValidationsGroupsEnum.default] })` above the `@IsNumberString...` here and below? As well as on senior housing, availability, and zipcode?

Upstream just merged changes that seem to make this validation work correctly, so I think we'll need to actually allow these to be optional now.

Reply:
Done